docs/document/content/user-manual/shardingsphere-proxy/cdc/build.cn.md

```diff
@@ -0,0 +1,160 @@
+
+### 配置 GLT 模块（可选）
+
+官网发布的二进制包默认不包含 GLT 模块，不保证跨库事务完整性，如果使用的是包含 GLT 功能的 openGauss 数据库，则可以额外引入 GLT 模块，保证跨库事务的完整性。
+
+目前有两种方式引入 GLT 模块，并且需要在 server.yaml 中也进行相应的配置。
+
+#### 1. 源码编译安装
+
+1. 准备代码环境，提前下载或者使用 Git clone，从 Github 下载 [ShardingSphere](https://github.com/apache/shardingsphere.git) 源码。
+2. 删除 kernel/global-clock/type/tso/core/pom.xml 中 shardingsphere-global-clock-tso-provider-redis 依赖的 `<scope>provided</scope>` 标签和 kernel/global-clock/type/tso/provider/redis/pom.xml 中 jedis 的 `<scope>provided</scope>` 标签 
+3. 编译 ShardingSphere-Proxy，具体编译步骤请参考 [ShardingSphere 编译手册](https://github.com/apache/shardingsphere/wiki#build-apache-shardingsphere)。
+
+#### 2. 直接引入 GLT 依赖
+
+可以从 maven 仓库中引入
+
+1. [shardingsphere-global-clock-tso-provider-redis](https://repo1.maven.org/maven2/org/apache/shardingsphere/shardingsphere-global-clock-tso-provider-redis)，下载和 ShardingSphere-Proxy 同名版本
+2. [jedis-4.3.1](https://repo1.maven.org/maven2/redis/clients/jedis/4.3.1/jedis-4.3.1.jar)
+
+### CDC Server 使用手册
+
+1. 修改配置文件 `conf/server.yaml`，打开 CDC 功能。 目前 `mode` 必须是 `Cluster`，需要提前启动对应的注册中心。如果 GLT provider 使用 Redis，需要提前启动 Redis。
+
+配置示例：
+
+```yaml
+mode:
+  type: Cluster
+  repository:
+    type: ZooKeeper
+    props:
+      namespace: open_cdc
+      server-lists: localhost:2181
+      retryIntervalMilliseconds: 500
+      timeToLiveSeconds: 60
+      maxRetries: 3
+      operationTimeoutMilliseconds: 500
+
+authority:
+  users:
+    # 这里的用户名和密码在 CDC Client 的认证中也会用到
+    - user: root@%
+      password: root
+    - user: proxy
+      password: Proxy@123
+  privilege:
+    type: ALL_PERMITTED
+
+# 开启 GLT 的时候也需要打开分布式事务
+#transaction:
+#  defaultType: XA
+#  providerType: Atomikos
+
+# GLT 模块配置，如果不需要 GLT 模块，可以不配置
+#globalClock:
+#  enabled: true
+#  type: TSO
+#  provider: redis
+#  props:
+#    host: 127.0.0.1
+#    port: 6379
+
+
+props:
+  cdc-server-port: 33071 # CDC Server 端口，必须配置
+  proxy-frontend-database-protocol-type: openGauss
+  # 省略其他配置
+  ......
+```
+
+2. 引入 JDBC 驱动。
+
+proxy 已包含 PostgreSQL JDBC 驱动。
+
+如果后端连接以下数据库，请下载相应 JDBC 驱动 jar 包，并将其放入 `${shardingsphere-proxy}/ext-lib` 目录。
+
+| 数据库       | JDBC 驱动                                                                                                                               | 参考                                                                                               |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| MySQL     | [mysql-connector-java-8.0.11.jar]( https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.49/mysql-connector-java-5.1.49.jar ) | [Connector/J Versions]( https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.11/mysql-connector-java-8.0.11.jar ) |
```

Comment:
1, Reference column could be removed

2, mysql-connector-java url could be updated

3, mysql-connector-java version should be the same as proxy